tools/train.py

```diff
@@ -91,12 +91,50 @@ def parse_args():
     return args
 
 
+def setup_multi_processes(cfg):
+    # set multi-process start method as `fork` to speed up the training
+    if platform.system() != 'Windows':
+        mp.set_start_method('fork')
+
+    # disable opencv multithreading to avoid overwheleming
+    cv2.setNumThreads(0)
```

Comment:
How about we make it settable through cfg? 
For now, cfg seems to be not used

Reply:
make sense